src/main/java/uk/gov/justice/digital/oasys/api/OffenderAliasDto.java

```diff
@@ -24,17 +23,24 @@
     @JsonProperty("forename2")
     private String forename2;
 
-    public static List<OffenderAlias> from(List<uk.gov.justice.digital.oasys.jpa.entity.OffenderAlias> aliases) {
-        return aliases.stream().map(OffenderAlias::from).collect(Collectors.toList());
+    public static List<OffenderAliasDto> from(List<uk.gov.justice.digital.oasys.jpa.entity.OffenderAlias> aliases) {
+        return aliases.stream().map(OffenderAliasDto::from).collect(Collectors.toList());
     }
 
-    private static OffenderAlias from(uk.gov.justice.digital.oasys.jpa.entity.OffenderAlias alias) {
-        return new OffenderAlias(
+    private static OffenderAliasDto from(uk.gov.justice.digital.oasys.jpa.entity.OffenderAlias alias) {
+        return new OffenderAliasDto(
                 alias.getOffenderAliasPk(),
-                TypesTransformer.localDateOf(alias.getAliasDateOfBirth()),
+                localDateOf(alias.getAliasDateOfBirth()),
                 alias.getAliasFamilyName(),
                 alias.getAliasForename1(),
                 alias.getAliasForename2());
     }
 
+    private static LocalDate localDateOf(Timestamp timestamp) {
```

Comment:
I think we might be able to just use LocalDateTime in the Entities. We seem to have a mix of sql Timestamp and LocalDateTime now.